src/main/scala/diplomacy/Nodes.scala

```diff
@@ -182,11 +184,21 @@ trait InwardNodeHandle[DI, UI, EI, BI <: Data] extends NoHandle
   def :*=*[EY](h: OutwardNodeHandle[DI, UI, EY, BI])(implicit p: Parameters, sourceInfo: SourceInfo): NoHandle = { bind(h, BIND_FLEX);  NoHandleObject }
 }
 
-sealed trait NodeBinding
-case object BIND_ONCE  extends NodeBinding
-case object BIND_QUERY extends NodeBinding
-case object BIND_STAR  extends NodeBinding
-case object BIND_FLEX  extends NodeBinding
+sealed trait NodeBinding {
+  def serialize: String
```

Comment:
I think if you called this method `toString` then it would just magically do the right thing when interpolated into a string without you even having to call `serialize` when you do so?